.github/workflows/release-series-images.yml

```diff
@@ -95,33 +87,22 @@ jobs:
         # which is required for build-push-action
         IFS=,
         echo "::set-output name=taglist::${tag_list[*]}"
-
+    
```

Comment:
Looks like this extra tab snuck in, can you rebase that out? See https://osg-htc.org/technology/software/git-software-development/#avoid-whitespace-noise